_episodes/12.md

```diff
@@ -0,0 +1,263 @@
+the [base.py](https://github.com/apache/superset/blob/master/superset/db_engine_specs/base.py)
+methods, but each engine, like Trino, will add the custom mappings via the
+[trino.py engine](https://github.com/apache/superset/blob/master/superset/db_engine_specs/trino.py)
+to verify that Superset and Trino have valid interactions.
+
+The main contributions in this pull request was adding a few custom changes for 
+Trino to work correctly with Superset. The main change applied was truncating
+the complex timestamps coming from Trino to a format that Superset is able to
+support during time aggregation operations.
+
+This along with some tests opened up a vast amount of functionality that we're
+now able to accomplish with Trino + Superset. We wanted to feature this because
+it goes to show how a small code change, even one that is not in the Trino
+repository, can have a vast effect on those using Superset and Trino.
+
+Thank you so much to [dungdm93](https://github.com/dungdm93) for making this
+change and further linking Trino into a fantastic project like [Apache
+ Superset](https://superset.apache.org/)!
+
+
+## PR Demo: Superset PR 13105 feat: first step native support Trino
+
+To put this PR to the test, we need to connect Superset to Trino as our
+datasource. 
+
+First, you need to follow [these instructions](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose)
+to install Docker (if you don't already have it installed), clone the superset.
+
+`$ git clone https://github.com/apache/superset.git`
+
+Once you have cloned the repository, before you run docker-compose, you want to
+set up the database driver to download. To do this navigate to the root
+directory of the local Superset repository you just downloaded and run the
+following.
+
+`echo "sqlalchemy-trino" >> ./docker/requirements-local.txt`
+
+This tells Superset scripts to install the sqlalchemy-trino library upon
+startup. We know the name by looking up [the Trino driver page](https://superset.apache.org/docs/databases/trino)
+for the driver documentation and how to use the connection string. Note: if you
+were to install these directly on a Superset node, you would refer to [this
+database drivers page](https://superset.apache.org/docs/databases/installing-database-drivers).
+
+
+Once this is done run the following to start up Superset and make sure you're in
+the root folder of the repo.
+
+`docker-compose -f docker-compose-non-dev.yml up`.
+
+Once Superset is up you will also want to bring up a Trino instance. We did so
+using a seperate docker-compose app. 
+
+As soon as this is done, you can navigate to Superset's homepage <http://localhost:8088>
+and scroll to the Data > Databases menu. 
+
+Click the + Database button.
+
+Set the Name to "Trino" and the URI to `trino://trino@host.docker.internal:8080`
+and click "Add".
+
+Note: If you want to allow CTAS, CVAS, or DML operations, you'll want to edit
```

Comment:
Remove Note: and update

If you want to allow CTAS, CVAS, or DML operations, edit the database, and click on ... 

If